build.gradle.kts

```diff
@@ -0,0 +1,88 @@
+import org.jetbrains.kotlin.gradle.dsl.JvmTarget
+import org.jetbrains.kotlin.gradle.plugin.KotlinBasePluginWrapper
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+import java.time.Duration
+
+plugins {
+    id("java")
+    id("org.jetbrains.kotlin.jvm") version "2.1.10"
+    id("org.jlleitschuh.gradle.ktlint") version "12.1.2"
+    id("io.github.gradle-nexus.publish-plugin") version "2.0.0"
+}
+
+group = project.property("GROUP_ID") as String
+
+apply("gradle-tasks/specs.gradle.kts")
```

Comment:
$rootDir for both?

Reply:
Not needed I believe.

Reply:
So remove from both?

Reply:
For better consistency and safety, I made sure to add the `$rootDir` prefix to the path. While it is unnecessary in practice, it would be nice to ensure the path is always valid instead of depending on gradle to resolve it.